### ISSUE TITLE
[Feature request] - Cloudflare Analytics

### ISSUE THREAD

**Issue by orester** [cloudflare/cloudflare-grafana-app#2]
It would be really nice if we were able to use this app also for Cloudflare Analytics (Requests, Bandwith, Visitors, ..). Is there any plan for broaden functionality for other cloudflare products?

Thx & WBR
Oliver

**Comment by danigrant** [comment]
Yeah! Over time we want to add more Cloudflare analytics like threats and traffic to the Grafana plugin.

**Comment by orester** [comment]
Many Thx Dani &  Really happy to read that :)

**Comment by sflaster** [comment]
This plugin is great, thanks for the heavy lifting and can't wait for the Zone Analytics !

**Comment by orester** [comment]
Hi,

any plans for Cloudflare Analytics?

BR
Oliver

**Comment by danigrant** [comment]
Not yet - we'll let you know!

**Comment by wowi42** [comment]
Hi @danigrant,
I'm ready to help (or a guy from my team), but there is a small issue: number of allowed requests + the fact that you need to do one call per zone on the analytics API.

We are already having a prototype with a cache layer, but I think it's very sad to do it like this. Any ideas? Event Socket? increase the limit?

**Comment by mty22** [comment]
@danigrant FYI, I worked with your support team in CF Support ticket #1374141

Used quite a bit of my time coordinating, and the fix was quite simple when I had a look at the plugin it's self.

**Comment by orester** [comment]
Hi,

any progress? It would be sooo nice and brilliant to have requests, bandwith, unique visitors, threats, status codes and specially ARGO  in our internal monitoring :)

BR
Oliver

**Comment by elithrar** [comment]
No planned work at the moment, Oliver (happy to take pull requests!)

**Comment by JorgenPhi** [comment]
I wrote a simple cron script to port Cloudflare Analytics to an InfluxDB database, which is supported with Grafana. Feel free to check it out and let me know what new features / metrics you would like to see.

https://github.com/JorgenPhi/cloudflare2db

**Comment by lgwapnitskyTN** [comment]
Any update on this?  Thank you

**Comment by gustavoserafim** [comment]
Any eta for this already?

**Comment by sflaster** [comment]
I would also love to see Cloudflare Web Analytics into Grafana with the same ease we added the DNS Analytics.

**Comment by ekifox** [comment]
Up :)
